lecture03/List_Map/src/UnKnownPokemonException.java

```diff
@@ -0,0 +1,16 @@
+public class UnKnownPokemonException extends Exception{
+    private String code;     //独自エラーコード
+    private String message;  //独自エラーメッセージ
+    UnKnownPokemonException(String code, String message) {
```

Comment:
どこかでゆーだいさんの指摘をみたので既出とおもいますが、改行がほしいです。
```suggestion
    private String message;  //独自エラーメッセージ

    UnKnownPokemonException(String code, String message) {
```

Reply:
こちらのコミットで対応しております。
[4524ca0](https://github.com/aoha968/Java2211/pull/3/commits/4524ca0b5630b1e3c17efd39cbd4bd7b4031409d)